Dockerfile

```diff
@@ -46,8 +35,9 @@ RUN mkdir /run/php \
 ENV TORAN_PROXY_VERSION 1.5.4
 
 # Download Toran Proxy
-RUN curl -sL https://toranproxy.com/releases/toran-proxy-v${TORAN_PROXY_VERSION}.tgz | tar xzC /tmp \
-    && mv /tmp/toran /var/www
+RUN mkdir /scratch/
+RUN curl -sL https://toranproxy.com/releases/toran-proxy-v${TORAN_PROXY_VERSION}.tgz | tar xzC /scratch \
+    && mv /scratch/toran/* /var/www
```

Comment:
Why use `/scratch` instead of `/tmp` ?

Reply:
The real trick is the /* , without it the code does not work even before my fork.

I notice that you pushed something to dockerhub manually, after the current latest master which probably did something similar.

I was hesitant to just copy /tmp/* so added a /tmp/scratch/* to ensure it's exactly what we extracted.

Reply:
Ok, `/tmp/toran/*` is not enough ?

I prefer keep directory /tmp to respect linux standards

Can you add `mkdir -p /var/www` before `mv` to ensure `/var/www` exist please ?

Reply:
Ok, will do! 👍 

BTW, we can try 20.04 and php-7.4 (which is default with 20.04)
but I was worried about breaking changes from 7.1 to 7.4 which I don't have capacity to deal with if so. (right now anyway)

Reply:
Ok thank you.

Yes! Maybe merge this pull request to PHP 7.2 and next open a new to try 7.4 ?

Reply:
Ok. 
mkdir -p /var/www (wont fail if it does exist)
/tmp/toran

And a missing fix for 7.2.

Should be good for merge, then I'll create the 20.04 PR later